src/emulator/app/src/app_init.cpp

```diff
@@ -268,6 +268,16 @@ bool init(HostState &state, Config cfg, const Root &root_paths) {
         }
     }
 
+    // Workaround driver bugs
+    const std::string gpu_name = reinterpret_cast<const GLchar*>(glGetString(GL_RENDERER));
+
+    if (gpu_name.find("GeForce RTX") != std::string::npos || gpu_name.find("RTX") != std::string::npos ||
```

Comment:
```suggestion
    if (gpu_name.find("GeForce RTX") != std::string::npos
```
you can remove all other, is like that name
![image](https://cdn.discordapp.com/attachments/442344583293304833/608796078942322701/unknown.png)